google-cloud-tools-plugin/src/com/google/gct/idea/appengine/cloud/DoManagedVmDeployment.java

```diff
@@ -63,6 +77,13 @@
   }
 
   public void run() {
+    final File appDefaultCredentialsPath = createApplicationDefaultCredentials();
+    if (appDefaultCredentialsPath == null) {
+      callback.errorOccurred(
+              GctBundle.message("appengine.deployment.credential.not.found",
```

Comment:
oh yea weird - when I ran the deploy forcing this exception I saw {0} printed on the command line view. I'll check again.